src/main/client/src/common/modals/DeleteBookModal.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+import { Button, Header, Icon, Modal } from 'semantic-ui-react';
+
+export default class DeleteBookModal extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            open: false
+        };
+        this.close = this.close.bind(this);
+    }
+
+    close = () => {
```

Comment:
Need to make a code style decision, this function is not declared in the same way as others in the project